### DISCUSSION TITLE
paire style problem

### DISCUSSION THREAD

**Discussion by WU2JING** [deepmodeling/deepmd-kit#522] [General]
Hello everyone, when I try to use three deep potential models defining paire_style in lammps. However, I encountered an error, as shown in the figure. I am stuck in the problem for a long time.
Any discussion is welcomed. Thank you.
   
![error](https://user-images.githubusercontent.com/68996784/114998406-831dfa00-9ed3-11eb-9a92-a8a8fff22323.PNG)
![potential](https://user-images.githubusercontent.com/68996784/114999140-2a029600-9ed4-11eb-9f56-fa5e87b99090.PNG)

**Comment by amcadmus** [comment]
The suggested option is to put the training data of BAsGaN, BAs and GaN together, and train a DP model out of them.

The hybridization of models does not really make sense.

**Reply by WU2JING** [reply]
Thank you.
